client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -19,9 +19,7 @@ registerBlockType( 'jetpack/simple-payments', {
 
 	category: 'jetpack',
 
-	keywords: [
-		/** @TODO add keywords */
-	],
+	keywords: [ 'simple payments' ],
```

Comment:
"Paypal" seems like a term people could use to look for this block as well since that's the only payment processor currently?

Reply:
Adding PayPal is a great idea!